frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BasePullBranchFastForwardOnlyAction.java

```diff
@@ -99,6 +101,10 @@ public void actionPerformed(AnActionEvent anActionEvent) {
       }
 
       if (FetchUpToDateTimeoutStatus.isUpToDate(gitRepository)) {
+        VcsNotifier.getInstance(project).notifyInfo(/* displayId */ null,
```

Comment:
<img width="385" alt="image" src="https://user-images.githubusercontent.com/19799111/168777412-21c70837-31ff-4d11-b6df-a91bdb73969f.png">

we've got a little situation here Xd

Our pull (which is fast-forward merge underhood) sends a notification too (and it was first 🏅 !).
So what happens here is that within a minute after fetch...
1. pull action triggered
2. no fetch vcs notification send
3. fast forward merge notification send

and in result, no fetch notification shows only for a very short time (one cannot read it), which looks bad. 

we can:
1. show the no fetch notification somewhere else (not with vcs notifier)
2. merge notification messages in case we need both of them. smth like: `No fetch blah blah. Fast forwarded blah blah` 

In the case of the first option, we could use a standard notification with a linked action that would enforce a fetch - seems nice to me.